v2/cli/init_command.go

```diff
@@ -0,0 +1,149 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"text/template"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/urfave/cli/v2"
+)
+
+type InitFields struct {
+	ID       string
+	Language string
+	Git      string
+}
+
```

Comment:
This type is only used in one place, for one very specific purpose, so let's move it closer to where it's used-- down around what is now line 82-- and yes, you can define types inside a function. Those types can only be used _within_ said function, but that's just fine.